packages/allow-scripts/src/cli.js

```diff
@@ -32,7 +32,7 @@ async function start () {
     }
     case 'setup': {
       writeRcFile()
-      addPreinstallAFDependency()
+      patchPackageJson(existsSync('./.npmrc') ? 'npm' : 'yarn1')
```

Comment:
we have a much better implementation of this and if we're in need of it here, we should use that.
I thought we'd eliminate the install step here and let the user run whatever installation they like.
(note there's a lot of options. `npm install` or `npm ci`, `yarn` alone or with options to strictly adhere to lockfile etc.)

Reply:
Ah, the main idea here was to retain previous behavior of the allow-scripts cli - either way, this logic is now removed.